packages/blockfrost/test/e2e/queryTransactions.test.ts

```diff
@@ -26,5 +26,115 @@ describe('blockfrostWalletProvider', () => {
         txId: '2db6592c4782064295295b365c2e8ce84084fa24b1b3f5834f3c6b65268b7878'
       });
     });
+    // TODO: waiting for fix in db-sync/blockfrost
+    // it('has collaterals for failed contract', async () => {
+    //   const [tx] = await walletProvider.queryTransactionsByHashes([
+    //     Cardano.TransactionId('9298f499a4c4aeba53a984cb4df0f9a93b7d158da4c2c2d12a06530841f94cd7')
+    //   ]);
+    //   expect(tx.body.inputs!.length).toEqual(0);
+    //   expect(tx.body.outputs!.length).toEqual(0);
+    //   expect(tx.body.collaterals!.length).toEqual(1);
+    // });
+    it('has withdrawals', async () => {
+      const [tx] = await walletProvider.queryTransactionsByHashes([
+        Cardano.TransactionId('e92e3c2ce94d61449876e23ba9170ca20868ee447f13703c5fa7e888cc1701e1')
+      ]);
+      expect(tx.body.withdrawals).toEqual([
+        { quantity: 29_308_336n, stakeAddress: 'stake_test1urlhkh2pl2xt24dkgjtqrkzfv77ekqj950znqpzdsz2wuds0xlsk6' }
+      ]);
+    });
+    it('has redeemer', async () => {
+      const [tx] = await walletProvider.queryTransactionsByHashes([
+        Cardano.TransactionId('3353309c6d3f2200c4e2084f76c1f1495f00eb21ea62f29c01da2adac71e1068')
+      ]);
+      expect(tx.witness.redeemers).toEqual([
+        {
+          executionUnits: { memory: 555_670, steps: 229_163_102 },
+          index: 0,
+          purpose: 'mint',
+          scriptHash: '87c822cd8fb44f2e3bffc3eaf41c63c2301a0ac2325ee3db634bd435'
+        }
+      ]);
+    });
+    // TODO: not implemented
```

Comment:
Please elaborate on why this isn't implemented

Reply:
No particular reason. There is a task in backlog for this issue.